### ISSUE TITLE
Gesture conflict on Android

### ISSUE THREAD

**Issue by fobos531** [nandorojo/zeego#23]
This is basically a duplicate of https://github.com/mpiannucci/react-native-context-menu-view/issues/43

I determined that the same issue present in this library - which means the android native implementations are most likely the same.  It might be a good idea to include the information present in the issue above in the docs whenever they are released.

**Comment by nandorojo** [comment]
Good call. I’ll leave this open until the docs are released and mention it there.

**Comment by xmflsct** [comment]
@fobos531 is this the behaviour you are describing? https://github.com/tooot-app/app/issues/676

**Comment by fobos531** [comment]
@xmflsct Kind of, yeah
